### DISCUSSION TITLE
Would Taskwiki word with Taskwarrior 3.0

### DISCUSSION THREAD

**Discussion by newptcai** [tbabej/taskwiki#444] [General]
Recently Taskwarrior released 3.0 version. I wonder if we will be able to use it?

**Comment by cameronj86** [comment]
Did you ever find this out? Looking to use it but unsure if it still works

**Comment by newptcai** [comment]
I have never tried. I use gitlab to synchronize my VimWiki across devices. TaskWarrior 3 switches to use sqlite as database, which does not really work well with git.

**Comment by cameronj86** [comment]
Thanks. I checked the [Taskwarrrior tools](https://taskwarrior.org/tools/) list and it is listed for both V2 & V3 so that answers my question.
